projects/jetstack/cert-manager/helm/Chart.yaml.template

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
```

Comment:
Will these removed now that you are cloning?

Reply:
Yeh, I am going to remove these templates and replace them with a search and replace thing.